docs/api/filters.rst

```diff
@@ -10,4 +13,104 @@ Filters
    filters/morphological_filters
    filters/morphological_filters_functional
 
-.. toctree::
\ No newline at end of file
+.. toctree::
+
+Filters in ``histolab`` are designed to be applied singularly or combined in a chain of transformations. A composition of filters is predefined for tissue segmentation, while custom filter combinations can be used for tissue detection or other tasks.
+
+* Image filters:
```

Comment:
this is useful but too wall text, we should find a way to present the same information in a cleaner way.. maybe we could add images?

Reply:
Do u have suggestions? anyway I agree with images, we have plenty of them